website/static/js/pages/file-page.js

```diff
@@ -4,4 +4,42 @@ var $osf = require('js/osfHelpers');
 var FileViewPage = require('js/filepage');
 var waterbutler = require('js/waterbutler');
 
+require('jquery-tagsinput');
+
 m.mount(document.getElementsByClassName('file-view-panels')[0], FileViewPage(window.contextVars));
+
+$(function() {
+    // Tag input
+    $('#fileTags').tagsInput({
+        width: '100%',
+        interactive: window.contextVars.currentUser.canEdit,
+        maxChars: 128,
+        onAddTag: function (tag) {
+            var url = '/api/v1/project/' + window.contextVars.node.id + '/osfstorage' + window.contextVars.file.path +
```

Comment:
If you must write URLs this way (such as if the page handles multiple files and needs to generate URLs dynamically for each), at least consider using the templating feature of a library such as URIjs:

https://medialize.github.io/URI.js/
(templating feature at bottom of page)